pkg/kyamls/helpers.go

```diff
@@ -30,11 +30,10 @@ func GetNamespace(node *yaml.RNode, path string) string {
 	return GetStringField(node, path, "metadata", "namespace")
 }
 
-// GetLabels returns the labels for the given file
-func GetLabels(node *yaml.RNode, path string) (map[string]string, error) {
-	labels, err := node.Pipe(yaml.Lookup("metadata", "labels"))
+func getMetadataMap(node *yaml.RNode, path string, mapName string) (map[string]string, error) {
+	labels, err := node.Pipe(yaml.Lookup("metadata", mapName))
```

Comment:
Can we call it metadata (or something else) instead of labels, as it now returns metadata and labels based on mapName?

Reply:
Sure.

Reply:
Done.